lib/css/modules/_showImages.scss

```diff
@@ -285,22 +176,150 @@ div.madeVisible {
 			content: '\F178';
 		}
 	}
-}
 
-.res-player {
-	position: absolute;
+	.res-media-rotatable & {
+		opacity: 0;
+	}
+
+	.res-media-rotatable:hover & {
+		opacity: .8;
+		transition: opacity 300ms;
+	}
 
-	// Hide "lookup" button on video's
-	.RESMediaControls .imageLookup {
+	.res-media-load-error ~ & {
 		display: none;
 	}
 }
 
-// video-advanced
+.res-expando {
+	display: block;
+
+	.usertext-body {
+		user-select: text; // Allow selection if is a text expando
+	}
+
+	.res-expando-media {
+		display: inline-block;
```

Comment:
Can you add
```scss
> * {
    display: inline-block;
}
```
immediately below this line?

(The `.res-image` and `.video-advanced` containers still end up with `block` sizing, so they fill the full width, which can interfere with middle-click scrolling -- I know, that's a *very* niche use-case)

If you think that selector is too broad and would cause problems, feel free to refine it, but I didn't notice any.